### DISCUSSION TITLE
Group - flex-grow

### DISCUSSION THREAD

**Discussion by cstrat** [mantinedev/mantine#97] [Q&A] [answered]
I am using `<Group>` with `grow` enabeld and on child elements assigning `flex-grow` values to adjust ratios. For example I have a form with three fields, the first is large the last two are small, so I have set `style={{ flexGrow: 5 }}` on the first child.
The docs say that with `grow` enabled, flex-grow is set to 1.

Just checking that relying on the style override here is not going to be an issue long term?

**Comment by rtivital** [comment, marked as answer]
Group children always have `flex-grow` either 0 or 1, it's a simple pattern to stretch content to 100% of the container width, to do more complex stuff with flexbox it will always be much easier to use your own flex container or Grid component.

**Reply by cstrat** [reply]
Yeah that does make sense.
